### PR TITLE
[doc] add readme in cn

### PR DESCRIPTION
**Pull request by benja-wu**

### CODE REVIEW

**Review by megaeasex — APPROVED** [review]
[TASK:easegress-pr-test SUCCESS]megaease/easegress Pull Request 267 Deploy Test Success

File: README.zh-CN.md [hunk 1]
```diff
@@ -0,0 +1,318 @@
...
+## 什么是Easegress
+
+`Easegress`是一个云原生流量协调系统，旨在。
+
+- **高可用性：** 内置Raft共识和领导者选举提供99.99%的可用性。
+- **流量编排：** 为每个流量管道简单编排各种过滤器。
+- **高性能：** 轻量级和基本功能加快了性能。
+- **可观察性：** 有许多有意义的统计数据定期以可读的方式进行。
+- **可扩展性：** 很容易用高级编程语言开发自己的过滤器或控制器。
+- **集成性：** 简单的接口使它很容易与其他系统集成，如Kubernetes Ingress、[EaseMesh](https://github.com/megaease/easemesh) 边车、工作流等。
```
Comment (localvar) [inline README.zh-CN.md:30]:
```suggestion
## Easegress 简介

`Easegress`是一个云原生流量协调系统，旨在提供：

- **高可用性：** 内置Raft共识和选主算法，提供99.99%的可用性。
- **流量编排：** 支持多种流量过滤器，轻松编排流量处理流程（Pipeline）。
- **高性能：** 基础功能采用轻量级方法实现，性能优异。
- **可观察性：** 周期性报告多种统计数据，系统状态尽在掌握。
- **可扩展性：** 良好的 API 设计，不必知道底层细节，也能自己开发过滤器和控制器。
- **集成性：** 接口简单，易于与其他系统集成，如Kubernetes Ingress、[EaseMesh](https://github.com/megaease/easemesh) 边车、工作流等。
```

File: README.zh-CN.md [hunk 2]
```diff
@@ -0,0 +1,318 @@
...
+		- HTTP/1.1
+		- HTTP/2
+		- HTTP/3(QUIC)
+		- MQTT(即将推出)
+	- **丰富的路由规则：** 精确路径、路径前缀、路径的正则表达式、方法、头文件。
+	- **弹性和容错**。
+		- **断路器：** 暂时阻止可能的故障。
+		- **速率限制器：** 限制传入请求的速率。
+		- **重播器：** 重复失败的执行。
+		- 时间限制器：** 限制执行的时间。
```
Comment (jxd134) [inline README.zh-CN.md:49]:
```suggestion
		- **时间限制器：** 限制执行的时间。
```

File: README.zh-CN.md [hunk 3]
```diff
@@ -0,0 +1,318 @@
...
+	- **部署管理**
+		- **蓝绿策略：** 一次性切换流量。
+		- **金丝雀策略：** 按着色安排流量。
+	- **API管理**
+		- **API聚合：** 聚合多个API的结果。
+		- **API协调：** 协调API的流动。
+	- **安全**
+		- **IP过滤：** 限制对IP地址的访问。
```
Comment (jxd134) [inline README.zh-CN.md:57]:
```suggestion
		- **IP过滤：** 限制对IP地址/地址段的访问。
```

File: README.zh-CN.md [hunk 4]
```diff
@@ -0,0 +1,318 @@
...
+		- **静态HTTPS：** 静态证书文件。
+		- **API签名：** 支持[HMAC]（https://en.wikipedia.org/wiki/HMAC）验证。
+		- **JWT Verification:** 验证[JWT Token](https://jwt.io/)。
+		- **OAuth2:** 验证[OAuth/2](https://datatracker.ietf.org/doc/html/rfc6749)请求。
+		- **Let's Encrypt:** 自动管理证书文件。
+	- **管道过滤机制**。
+		- **责任链模式：** 协调过滤器链。
+		- **过滤器管理：** 使开发新过滤器变得容易。
+	- **服务网格**
+		- 网格主控：** 是管理网格服务生命周期的控制平面。
```
Comment (jxd134) [inline README.zh-CN.md:67]:
```suggestion
		- ** 网格主控：** 是管理网格服务生命周期的控制平面。
```

File: README.zh-CN.md [hunk 5]
```diff
@@ -0,0 +1,318 @@
...
+		- **网格入口控制器：** 是针对网格的入口控制器，将外部流量路由到网格服务。
+		  > 注意，该功能被[EaseMesh](https://github.com/megaease/easemesh)所使用。
+	- **第三方的集成**
+		- **FaaS** 与无服务器平台Knative集成。
+		- **服务发现** 与Eureka、Consul、Etcd和Zookeeper集成。
+		- **入站控制器** 与Kubernetes集成，作为入站控制器。
+- **扩展性**
+    - **WebAssembly**执行用户开发的[WebAssembly](https://webassembly.org/)代码。
+- **高性能和可用性**
+	- **适应性**：在处理链中适应请求和响应。
```
Comment (jxd134) [inline README.zh-CN.md:78]: 适用性，无法让人直观理解其作用。
是指httpTemplate吗？

File: README.zh-CN.md [hunk 6]
```diff
@@ -0,0 +1,318 @@
...
+			- **数据大小：** 请求和响应大小。
+			- **状态代码:** HTTP状态代码。
+			- **TopN:** 按聚合的API排序（仅在服务器维度）。
+
+
+## 用户案例
+
+下面的例子展示了如何在不同场景下使用Easegress。
+
+- [API Aggregator](./doc/cookbook/api_aggregator.md) - 将许多 API 聚集成一个 API。
```
Comment (jxd134) [inline README.zh-CN.md:103]:
```suggestion
- [API Aggregator](./doc/cookbook/api_aggregator.md) - 将业务关联 API  聚合为单个 API。
```

File: README.zh-CN.md [hunk 7]
```diff
@@ -0,0 +1,318 @@
...
+- [LoadBalancer](./doc/cookbook/load_balancer.md) - 负载均衡的若干策略 
+- [Distributed Tracing](./doc/cookbook/distributed_tracing.md) - 如何进行APM追踪 - Zipkin。
+- Kubernetes入口控制器](./doc/cookbook/k8s_ingress_controller.md) - 如何与Kubernetes集成作为入口控制器。
+- 性能](./doc/cookbook/performance.md) - 性能优化 - 压缩、缓存等。
+- [Pipeline](./doc/cookbook/pipeline.md) - 如何为请求/响应处理协调HTTP过滤器。
+- [Resilience and Fault Tolerance](./doc/cookbook/resilience.md) - 断路器、速率限制器、回流器、时间限制器等（移植自[Java resilience4j](https://github.com/resilience4j/resilience4j)
+- [Security](./doc/cookbook/security.md) - 如何通过Header、JWT、HMAC、OAuth2等进行认证。
+- [Service Proxy](./doc/cookbook/service_proxy.md) - 支持微服务注册表 - Zookeeper、Eureka、Consul、Nacos等。
+- [WebAssembly](./doc/cookbook/wasm.md) - 使用AssemblyScript来扩展Easegress
+- [Workflow](./doc/cookbook/workflow.md) - 一个为一些API制作工作流程的例子。
```
Comment (jxd134) [inline README.zh-CN.md:115]:
```suggestion
- [Workflow](./doc/cookbook/workflow.md) - 将若干 API 进行组合，定制为工作流。
```

File: README.zh-CN.md [hunk 8]
```diff
@@ -0,0 +1,318 @@
...
+完整的列表请参见 [Cookbook](./doc/cookbook/README.md)。
+
+
+## 入门
+
+Easegress的基本通用用法是为后端服务器快速设置代理。我们把它分成多个简单的步骤来说明基本概念和操作。
+
+### 设置 Easegress
+
+我们可以从[发布页](https://github.com/megaease/easegress/releases)下载二进制文件。例如，我们使用linux版本。
```
Comment (jxd134) [inline README.zh-CN.md:126]:
```suggestion
我们可以从[发布页](https://github.com/megaease/easegress/releases)下载二进制文件。  
例如，我们使用linux版本。
```

File: README.zh-CN.md (hunk 5 above)
Comment (localvar) [reply README.zh-CN.md:78]: Thanks for pointing this out, this is a draft version that was mostly translated from the English version by AI, we are rendering the words.

File: README.zh-CN.md [hunk 9]
```diff
@@ -0,0 +1,318 @@
...
+2021-05-17T16:45:39.253+08:00	INFO	cluster/cluster.go:496	session is ready
+2021-05-17T16:45:39.253+08:00	INFO	api/api.go:96	api server running in localhost:2381
+2021-05-17T16:45:44.235+08:00	INFO	cluster/member.go:210	self ID changed from 0 to 689e371e88f78b6a
+2021-05-17T16:45:44.236+08:00	INFO	cluster/member.go:137	store clusterMembers: eg-default-name(689e371e88f78b6a)=http://localhost:2380
+2021-05-17T16:45:44.236+08:00	INFO	cluster/member.go:138	store knownMembers  : eg-default-name(689e371e88f78b6a)=http://localhost:2380
+```
+
+Makefile的默认目标是将两个二进制文件编译到`bin/`目录中。`bin/easegress-server`是服务器端的二进制文件，`bin/egctl`是客户端的二进制文件。我们可以把它添加到`$PATH`中，以简化下面的命令。
+
+我们可以运行`easegress-server`而不指定任何参数，它通过打开默认的2379、2380、2381端口启动自己。当然，我们可以在配置文件中改变它们，或者在`easegress-server --help`中解释清楚的命令参数。
```
Comment (jxd134) [inline README.zh-CN.md:161]:
```suggestion
我们可以运行`easegress-server`而不指定任何参数（服务默认依赖于2379、2380、2381端口）。  
> 我们可以在配置文件中更改默认端口，或者在命令行启动时指定相关参数（参数具体释义可通过执行`easegress-server --help`命令获取）。
```

File: README.zh-CN.md (hunk 5 above)
Comment (benja-wu) [reply README.zh-CN.md:78]: As @localvar had said, thanks for the correction.

Comment (jxd134) [reply README.zh-CN.md:78]: get it

**Review by megaeasex — APPROVED** [review]
[TASK:easegress-pr-test SUCCESS]megaease/easegress Pull Request 267 Deploy Test Success

**Review by megaeasex — CHANGES_REQUESTED** [review]
[TASK:easegress-pr-test FAILED]megaease/easegress Pull Request 267 Deploy Test failed

**Review by megaeasex — APPROVED** [review]
[TASK:easegress-pr-test SUCCESS]megaease/easegress Pull Request 267 Deploy Test Success

File: README.zh-CN.md [hunk 10]
```diff
@@ -0,0 +1,333 @@
...
+- [LoadBalancer](./doc/cookbook/load_balancer.md) - 各种负载均衡策略。 
+- [Distributed Tracing](./doc/cookbook/distributed_tracing.md) - 如何使用 Zipkin 进行 APM 追踪。
+- [Kubernetes入口控制器](./doc/cookbook/k8s_ingress_controller.md) - 如何作为入口控制器与 Kubernetes 集成。
+- [性能](./doc/cookbook/performance.md) - 性能优化，压缩、缓存等。
+- [Pipeline](./doc/cookbook/pipeline.md) - 如何编排 HTTP 过滤器来处理请求和应答。
+- [Resilience and Fault Tolerance](./doc/cookbook/resilience.md) - 断路器、速率限制、重试、时间限制等（移植自[Java resilience4j](https://github.com/resilience4j/resilience4j)
+- [Security](./doc/cookbook/security.md) - 如何通过标头、JWT、HMAC、OAuth2 等进行认证。
+- [Service Proxy](./doc/cookbook/service_proxy.md) - 使用 Zookeeper、Eureka、Consul、Nacos 等进行服务注册。
+- [WebAssembly](./doc/cookbook/wasm.md) - 使用 AssemblyScript 来扩展 Easegress。
+- [Workflow](./doc/cookbook/workflow.md) - 将若干 API 进行组合，定制为工作流。
```
Comment (haoel) [inline README.zh-CN.md:116]:
```suggestion
- [API 聚合](./doc/cookbook/api_aggregator.md) - 将多个 API 聚合为一个。
- [函数即服务 FaaS](./doc/cookbook/faas.md) - 支持 Knative FaaS 集成。
- [高并发秒杀](./doc/cookbook/flash_sale.md) - 如何使用 Easegress 进行高并发的秒杀活动。
- [负载均衡](./doc/cookbook/load_balancer.md) - 各种负载均衡策略。 
- [分布式调用链](./doc/cookbook/distributed_tracing.md) - 如何使用 Zipkin 进行 APM 追踪。
- [Kubernetes入口控制器](./doc/cookbook/k8s_ingress_controller.md) - 如何作为入口控制器与 Kubernetes 集成。
- [高性能](./doc/cookbook/performance.md) - 性能优化，压缩、缓存等。
- [管道编排](./doc/cookbook/pipeline.md) - 如何编排 HTTP 过滤器来处理请求和应答。
- [弹力和容错设计](./doc/cookbook/resilience.md) - 断路器、速率限制、重试、时间限制等（移植自[Java resilience4j](https://github.com/resilience4j/resilience4j)
- [安全](./doc/cookbook/security.md) - 如何通过标头、JWT、HMAC、OAuth2 等进行认证。
- [服务网关](./doc/cookbook/service_proxy.md) - 使用 Zookeeper、Eureka、Consul、Nacos 等进行服务注册。
- [WebAssembly](./doc/cookbook/wasm.md) - 使用 AssemblyScript 来扩展 Easegress。
- [工作流](./doc/cookbook/workflow.md) - 将若干 API 进行组合，定制为工作流。
```

**Review by megaeasex — APPROVED** [review]
[TASK:easegress-pr-test SUCCESS]megaease/easegress Pull Request 267 Deploy Test Success

File: README.zh-CN.md [hunk 11]
```diff
@@ -0,0 +1,333 @@
...
+		- **节点**：角色（Leader、Writer、Reader）、健康状态、最后一次心跳时间，等等。
+		- **多维度的服务器和后端流量数据**
+			- **吞吐量**：请求数、TPS/m1、m5、m15 和错误百分比等。
+			- **延迟**：p25、p50、p75、p95、p98、p99、p999。
+			- **数据大小**：请求和响应大小。
+			- **状态代码**：HTTP状态代码。
+			- **TopN**：按 API 聚合并排序（仅服务器维度）。
+
+
+## 用户案例
```
Comment (haoel) [inline README.zh-CN.md:100]:
```suggestion
## 相关用例
```

**Review by megaeasex — APPROVED** [review]
[TASK:easegress-pr-test SUCCESS]megaease/easegress Pull Request 267 Deploy Test Success

**Review by xxx7xxxx — APPROVED** [review]

**Review by localvar — APPROVED** [review]

**Pull request merged**